src/amo/components/AddonReviewCard/styles.scss

```diff
@@ -76,7 +81,9 @@
 }
 
 .AddonReviewCard-control {
-  @include margin-end(6px);
+  display: flex;
+  max-width: 33%;
```

Comment:
I agree. I thought it might be hard but I copied and pasted this answer and it worked first try ✨ https://stackoverflow.com/questions/8720931/can-css-detect-the-number-of-children-an-element-has

Reply:
nice 👍